### ISSUE TITLE
404 on endpoints

### ISSUE THREAD

**Issue by eduardoltorres** [internetarchive/openlibrary#3879]
### Description

I went through the list of all routes listed on https://github.com/internetarchive/openlibrary/wiki/Endpoints#list-of-all-routes and hit some 404. Is this expected?

These are the relevant routes with a few comments:
```
/account/register      # should it be /account/create rather?
/account/preferences/template_preferences
/stats/waitinglists
/theme                   # all sub starting with /theme
/availability
/account/email
/account/password
/search/edition
/subjects              # this returns successfully on prod (https://openlibrary.org/subjects)
/borrow/about
/libraries               # all sub starting with /libraries
```

### Steps to Reproduce

1. Go to `localhost:8080/<endpoint>`

* Actual: 404
* Expected: 200 OK

### Details

- **Logged in (Y/N)?** both
- **Browser type/version?** Firefox 81.0 (64-bit)
- **Operating system?** Arch
- **Environment (prod/dev/local)?** local and prod

### Proposal & Constraints

I found the discussion https://github.com/internetarchive/openlibrary/pull/1211. Not sure if it means that https://dev.openlibrary.org/developers/routes should be used instead of https://github.com/internetarchive/openlibrary/wiki/Endpoints#list-of-all-routes.

- Replaced the endpoints with the correct ones?
- Remove them?
- I am misunderstanding docs maybe?

### Stakeholders

Sorry, not sure who this should be according to the leads on the wiki. Policy - @mekarpeles ?

**Comment by navneetsaluja** [comment]
If I remember correctly, Mek did say in the [technical walkthrough video](https://archive.org/details/openlibrary-tour-2020/technical_overview.mp4) that this list of endpoints wasn't updated and you might find a few errors. Other than that, I'd asked Drini about the /subjects endpoint, apparently the code for it lives in the production databases (not in Git) and you can only test it by creating a new page locally with the code from [here](http://openlibrary.org/subjects.json).

**Comment by eduardoltorres** [comment]
Yes, precisely. IIRC as well, he mentions that for those new to OL, it would be a good starting point to check the endpoints out and open an issue for potentially updating the page.

**Comment by jimchamp** [comment]
This documentation was recently updated (see #10433), so I believe that this can be closed.
